src/assistant/assistant-helper.js

```diff
@@ -136,14 +136,23 @@ const runBlockAssistant = async ({url, name, prompt, outputPath = DEFAULT_IMPORT
 const runCellAssistant = async ({url, name, prompt, outputPath = DEFAULT_IMPORTER_PATH}) => {
   const startTime = Date.now();
   const builder = await getBuilder(url, {useExisting:  true, outputPath});
-  const manifest = await builder.addCells(name, prompt);
+  const manifest = await builder.addCellParser(name, prompt);
   writeManifestFiles(manifest, outputPath);
   console.log(chalk.green(`${name} block parser generated successfully in ${getDurationText(startTime)}`));
 };
 
+const runPageAssistant = async ({url, name, prompt, outputPath = DEFAULT_IMPORTER_PATH}) => {
+  const startTime = Date.now();
+  const builder = await getBuilder(url, {useExisting:  true, outputPath});
```

Comment:
Picky - double space in there.  (Maybe fix if there are any more changes going in.)